prov/gni/src/gnix_fabric.c

```diff
@@ -636,8 +635,10 @@ static int _gnix_ep_getinfo(enum fi_ep_type ep_type, uint32_t version,
 					goto err;
 				}
 			} else {
+				/* FIXME: Check will not work with ofi_check_mr_mode
 				if (__gnix_check_mr_mode(mr_mode))
 					goto err;
+				*/
```

Comment:
@jswaro @hppritcha - I need to know what to do here

Reply:
I need to give it some thought. Let me get back to you.